javascript/javascript1/week2/homework/student-manager.js

```diff
@@ -0,0 +1,44 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName == null || studentName == "") {
```

Comment:
```suggestion
    if (studentName === null || studentName === "") {
```

Reply:
I'm afraid I have to disagree, as == covers more cases of possible invalid inputs e.g. studentName == null covers both undefined and null values.